### PR TITLE
CSV importer modified to check each file line. 

### PR DESCRIPTION
**Pull request by hcuffy**

### CODE REVIEW

**Comment by jtnelson** [comment]
@hcuffy can you rebase your changes against develop

```bash
$ ./update-fork.sh
```

File: concourse-integration-tests/src/test/resources/invalid.csv [hunk 1]
```diff
@@ -1 +1,2 @@
-< I'm the invalid test file >
```
Comment (jtnelson) [inline concourse-integration-tests/src/test/resources/invalid.csv]: any particular reason you changed the first line of the test file?

Comment (hcuffy) [inline concourse-integration-tests/src/test/resources/invalid.csv]: No... I didn't change it. If I did it was by accident.

Comment (hcuffy) [inline concourse-integration-tests/src/test/resources/invalid.csv]: Can't recall making that change to be honest.

Comment (jtnelson) [inline concourse-integration-tests/src/test/resources/invalid.csv]: Cool, can you just change it back to what it was and we should be good to merge

**Pull request closed without merging**